src/redux/wallets/actions.ts

```diff
@@ -511,6 +513,66 @@ export const updateTransactionFromBlockchain = (
     }
 };
 
+export const signAndSendTransaction = (tx: any, password: string, requestId: string) => async (
```

Comment:
why don't we use `sendTransferTransaction` action creator ? Your implementation will not work at it should with ledger hw wallet...

Reply:
also all the code from here seems to be duplicated with code from `sendTransferTransaction`